LICENSE

```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright {yyyy} {name of copyright owner}
+   Copyright 2016-2020 Istio Authors
```

Comment:
2022

Reply:
This is from https://github.com/istio/common-files/blob/master/files/LICENSE#L190. Would need to update it there. Again, this file gets copied to each repo as part of the `make update-common`.